content/hardware/02.hero/boards/uno-wifi-rev2/tutorials/uno-wifi-rev-2/uno-wifi-r2-chromebook-installation/uno-wifi-r2-chromebook-installation.md

```diff
@@ -0,0 +1,126 @@
+tags: 
+  - Chromebook
+  - Installation
+author: 'Karl Söderby'
+hardware:
+  - hardware/03.hero/boards/uno-wifi-rev-2
+software:
+  - web-editor
+---
+
+## Introduction 
+
+> This tutorial is only relevant for Chromebook users that uses an Arduino UNO WiFi Rev 2 board.  
+
+The [UNO WiFi Rev 2](https://store.arduino.cc/arduino-uno-wifi-rev2) is the connected version of the classic UNO board. If you are using a **Chromebook**, setting up your board is a bit different for this particular board:
+
+- You will need to upgrade the firmware using either a **Windows/Mac/Linux** computer, prior to programming it via a Chromebook. Detailed instructions are provided in this tutorial, and the process only takes a few minutes!
+- It is only possible to use the [Web Editor](https://create.arduino.cc/editor), an online IDE that is part of the [Arduino Cloud](https://cloud.arduino.cc/).
+
+***Note that only the Web Editor is supported in Chromebooks. It is not possible to configure and upload to UNO WiFi Rev 2 boards via the [IoT Cloud](https://create.arduino.cc/iot/things).***
+
+## Goals
+
+The goals of this project are:
+
+- Learn how to upgrade the firmware on your UNO WiFi Rev 2, so it can be used with a Chromebook.
+- Learn how to install the [Arduino Create for Education](https://chrome.google.com/webstore/detail/arduino-create-for-educat/elmgohdonjdampbcgefphnlchgocpaij) app from Chrome Web Store.
+- Learn how to upload a sketch to your board using a Chromebook and the Web Editor.
+
+## Hardware & Software Needed
+
+- [Arduino IDE 2](https://www.arduino.cc/en/software#future-version-of-the-arduino-ide). 
+- [Arduino Web Editor](https://create.arduino.cc/).
+- [Arduino Create for Education](https://chrome.google.com/webstore/detail/arduino-create-for-educat/elmgohdonjdampbcgefphnlchgocpaij) (Chrome Web Store)
+- [Arduino UNO WiFi Rev 2](https://store.arduino.cc/arduino-uno-wifi-rev2).
+
+***Please note that you need to install the Arduino IDE 2.0.X. The legacy IDE (1.8.X) will not work for this tutorial.***
+
+## Upgrading Firmware
+
+Since Chromebooks cannot run executables, the firmware upgrade for the UNO WiFi Rev 2 needs to be done through a computer with the Windows/Mac/Linux OS installed. 
+
+***While you won't be using the Arduino IDE 2.0 directly, you will be utilizing a set of tools that is included in the IDE. This is why you need to install the editor on your local machine.***
+
+**1.** Make sure you have installed [Arduino IDE 2.0.X](https://www.arduino.cc/en/software#future-version-of-the-arduino-ide). 
+
+**2.** Download the [optiboot_atmega4.hex](/resources/firmware/optiboot_atmega4809.hex) file, and move it to your **Desktop folder**. 
+
+![.hex file in your Desktop folder.](assets/hex-file-desktop.png)
+
+**3.** Open your Command Prompt (Windows) or Terminal (Mac), copy paste the command for your OS from the snippets below:
+
```

Comment:
I would add a comment saying that possibly these paths might be updated in the future, depending on the latest installed version. E.g. `6.3.0-arduino17` could become `6.Y.Z-arduinoX`.